pkg/api/azenvtypes.go

```diff
@@ -249,8 +249,8 @@ var (
 			Ubuntu:          SovereignCloudsUbuntuImageConfig,
 			RHEL:            DefaultRHELOSImageConfig,
 			CoreOS:          DefaultCoreOSImageConfig,
-			AKS:             SovereignCloudsUbuntuImageConfig,
-			AKSDockerEngine: SovereignCloudsUbuntuImageConfig,
+			AKS:             DefaultAKSOSImageConfig,
+			AKSDockerEngine: DefaultAKSDockerEngineOSImageConfig,
```

Comment:
@karataliu Are those images available now on Mooncake?

Reply:
They should be yes. Please test by building/deploying from this PR! :)

Reply:
Did you turn on MSI? I verified it can work without MSI.

Reply:
Yep, MSI enabled in my deployment